plugins/woocommerce/tests/e2e-pw/utils/site.js

```diff
@@ -0,0 +1,242 @@
+const api = require( './api' );
```

Comment:
Added a new file that contains utility functions for resetting/clearing the site from pre-existing test data.